styleguide/source/_patterns/01-molecules/08-lists/12-list-topic-tools/_list-topic-tools.scss

```diff
@@ -40,27 +51,44 @@
 
   // Topic tools list file icon.
   .list_topic-tools_icon {
+    display: none;
     float: left;
     height: 40px;
     margin-right: 14px;
     vertical-align: -webkit-baseline-middle;
     width: 40px;
+
+    @include breakpoint($bp-med) {
+      display: inline-block;
+    }
   }
 
   // Topic tools list title.
   .list_topic-tools_title {
```

Comment:
Suggestion to make these `display: block;` so that the spacing tightens up a bit and the file type and size move to a new line (this makes the list look cleaner IMO, but try it out and see what you think)